kolibri/content/api.py

```diff
@@ -202,7 +199,17 @@ def descendants(self, request, **kwargs):
 
     @detail_route(methods=['get'])
     def ancestors(self, request, **kwargs):
-        return Response(self.get_object().get_ancestors().values('pk', 'title'))
+        cache_key = 'contentnode_ancestors_{db}_{pk}'.format(db=get_active_content_database(), pk=kwargs.get('pk'))
+
+        if cache.get(cache_key) is not None:
+            return Response(cache.get(cache_key))
```

Comment:
Would only change if the content database was updated - so put a 10 minute cache timeout to prevent having to do explicit invalidation.